### PR TITLE
[Uniswap V3 Merge] Style fixes

### PR DESCRIPTION
**Pull request by W3stside**
Part of #680 

Tackles broken styles that occurred post merge

## Testing
- just use the app and find things that look wrong and post here or make an issue linked to this

### CODE REVIEW

File: src/components/Modal/index.tsx [hunk 1]
```diff
@@ -82,6 +82,7 @@ interface ModalProps {
   minHeight?: number | false
   maxHeight?: number
   initialFocusRef?: React.RefObject<any>
+  className?: string
```
Comment (W3stside) [inline src/components/Modal/index.tsx:85]: to extend uniswap `Modal` component via styled

File: src/custom/components/SearchModal/CurrencySearch/CurrencySearchMod.tsx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+          <TYPE.main color={theme.text3} textAlign="center" mb="20px">
+            <Trans>No results found.</Trans>
+          </TYPE.main>
+        </Column>
+      )}
+      <Footer>
+        <Row justify="center">
+          <ButtonText onClick={showManageView} color={theme.primary1} className="list-token-manage-button">
+            {/* 
+            <RowFixed>
```
Comment (W3stside) [inline src/custom/components/SearchModal/CurrencySearch/CurrencySearchMod.tsx:231]: the mod

File: src/custom/pages/Swap/SwapMod.tsx [hunk 3]
```diff
@@ -4,10 +4,10 @@ import { Currency, CurrencyAmount, Token, TradeType } from '@uniswap/sdk-core'
 // import { Trade as V3Trade } from '@uniswap/v3-sdk'
 // import { AdvancedSwapDetails } from 'components/swap/AdvancedSwapDetails'
 import UnsupportedCurrencyFooter from 'components/swap/UnsupportedCurrencyFooter'
-// import { MouseoverTooltip, MouseoverTooltipContent } from 'components/Tooltip'
+import { MouseoverTooltip /* , MouseoverTooltipContent */ } from 'components/Tooltip'
 // import JSBI from 'jsbi'
 import React, { useCallback, useContext, useEffect, useMemo, useState } from 'react'
-import { ArrowDown /* , ArrowLeft, CheckCircle, HelpCircle, Info */ } from 'react-feather'
+import { ArrowDown /* , ArrowLeft, CheckCircle, HelpCircle, Info */, CheckCircle, HelpCircle } from 'react-feather'
```
Comment (W3stside) [inline src/custom/pages/Swap/SwapMod.tsx:10]:
```suggestion
import { ArrowDown, /*, ArrowLeft */ CheckCircle, HelpCircle /*, Info */ } from 'react-feather'
```

**Comment by alfetopito** [comment]
Account details is partially black
screenshot_2021-06-29_10-34-46

Light mode is ok

**Comment by alfetopito** [comment]
Confirm swap has no padding
screenshot_2021-06-29_10-36-15
And no text on dark mode at the bottom
screenshot_2021-06-29_10-36-53

**Comment by alfetopito** [comment]
Basically all modals have a black background on dark mode and no padding on both
screenshot_2021-06-29_10-38-17
screenshot_2021-06-29_10-38-08

**Comment by alfetopito** [comment]
Terms of service on connect wallet are hard to read AND those are Uniswap's terms of service not ours
screenshot_2021-06-29_10-39-54

File: src/components/Modal/index.tsx (hunk 1 above)
Comment (alfetopito) [reply src/components/Modal/index.tsx:85]: Shouldn't this be on a Mod file?

File: src/components/SearchModal/CurrencySearchModal.tsx [hunk 4]
```diff
@@ -3,12 +3,19 @@ import React, { useCallback, useEffect, useState } from 'react'
...
-import { CurrencySearch } from './CurrencySearch'
+import { CurrencySearch } from 'components/SearchModal/CurrencySearch'
 import { ImportToken } from './ImportToken'
 import usePrevious from 'hooks/usePrevious'
 import Manage from './Manage'
 import { TokenList } from '@uniswap/token-lists'
 import { ImportList } from 'components/SearchModal/ImportList'
+import styled from 'styled-components/macro'
+
+export const GpModal = styled(Modal)`
```
Comment (alfetopito) [inline src/components/SearchModal/CurrencySearchModal.tsx:14]: Same, Mod file?

**Review by alfetopito — CHANGES_REQUESTED** [review]
Some points regarding the mod rules here, besides the styles that are still broken in the comments

File: src/components/Modal/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/Modal/index.tsx:85]: i made a judgement call not to

File: src/components/SearchModal/CurrencySearchModal.tsx (hunk 4 above)
Comment (W3stside) [reply src/components/SearchModal/CurrencySearchModal.tsx:14]: ah yes this should, thought i was in the mod here thanks

**Comment by alfetopito** [comment]
Couple of screens are still broken:

1. Wallets modal:
(connect)
screenshot_2021-07-02_09-27-10
(connecting)
screenshot_2021-07-02_09-28-54
(connected)
screenshot_2021-07-02_09-30-42
    - Text on dark mode
    - Empty header block
    - Dark mode modal background

2. Confirm modal:
screenshot_2021-07-02_09-29-45
    - Not sure this is the new design, but looks too compact. The button feels much smaller.

4. Expert mode swap
screenshot_2021-07-02_09-28-44
    - Arrow for recipient address color and position

5. Settings modal
screenshot_2021-07-02_09-28-20
    - Toggle button color on dark mode

6. Manage lists
screenshot_2021-07-02_09-27-55
screenshot_2021-07-02_09-27-41
    - Colors/inputs on both dark and light modes are off. Some stuff could be unrelated to the merge as Michel recently fixed it

File: src/custom/components/Settings/index.tsx [hunk 5]
```diff
@@ -4,9 +4,9 @@ import styled from 'styled-components'
 import { RowFixed } from 'components/Row'
 import SettingsMod, { StyledMenuButton, MenuFlyout, StyledMenuIcon, EmojiWrapper } from './SettingsMod'
 import { Percent } from '@uniswap/sdk-core'
-import { INITIAL_ALLOWED_SLIPPAGE_PERCENT } from 'constants/index'
+// import { INITIAL_ALLOWED_SLIPPAGE_PERCENT } from 'constants/index'
```
Comment (alfetopito) [inline src/custom/components/Settings/index.tsx:7]:
```suggestion
```

**Review by alfetopito — COMMENTED** [review]
Unless I'm mistaken, it looks like there are couple of changes done in the Mod files that we would otherwise do in the respective index files instead.

I understand this is a huge, but I think we should stick to the rules to keep the code consistent with other parts

**Comment by W3stside** [comment]
> Unless I'm mistaken, it looks like there are couple of changes done in the Mod files that we would otherwise do in the respective index files instead.
> 
> I understand this is a huge, but I think we should stick to the rules to keep the code consistent with other parts

Some of that is unavoidable and/or old mods we did in the beginning before we had a clearer mod rule. Do you have any examples?

File: src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx [hunk 6]
```diff
@@ -27,6 +27,13 @@ import TradeGp from 'state/swap/TradeGp'
 import { INPUT_OUTPUT_EXPLANATION } from 'constants/index'
 import { computeSlippageAdjustedAmounts } from 'utils/prices'
 import { Field } from 'state/swap/actions'
+import { darken } from 'polished'
+
+// MOD
+const LightCard = styled(LightCardUni)`
```
Comment (alfetopito) [inline src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx:33]: Example of mod changes.

I would expect these to be in the index, and the parent component to take this custom component as a prop.

File: src/custom/components/AccountDetails/index.tsx [hunk 7]
```diff
@@ -176,7 +176,7 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName(connector, walletInfo)}
                 <div>
-                  {connector !== injected && connector !== walletlink && (
+                  {/* connector !== injected && connector !== walletlink && (
```
Comment (alfetopito) [inline src/custom/components/AccountDetails/index.tsx:179]: In this case (not a Mod), better remove it completely.
Unless you are commenting it out for some other reason?

File: src/custom/components/Settings/index.tsx (hunk 5 above)
Comment (alfetopito) [reply src/custom/components/Settings/index.tsx:7]: Any reason to keep this commented out here too?

**Review by alfetopito — APPROVED** [review]
:+1:

Loving the mods now :)

File: src/custom/components/AccountDetails/index.tsx (hunk 7 above)
Comment (W3stside) [reply src/custom/components/AccountDetails/index.tsx:179]: we may want it later

**Pull request merged**